WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -136,7 +136,13 @@ public void onResponse(JSONObject response) {
                         mRemoteSettings.localTableId = mBlog.getRemoteBlogId();
                         deserializeDotComRestResponse(mBlog, response);
                         if (!mRemoteSettings.equals(mSettings)) {
+                            // postFormats setting is not returned by this api call so copy it over
```

Comment:
Yeah, the empty list is not fixed by this PR (it happens without the PR as well).